### ISSUE TITLE
Github Packages not available

### ISSUE THREAD

**Issue by marchof** [microsoft/gctoolkit#63]
Hi, thanks for starting this project! I'm really looking forward to make use of it soon :)

It looks like the Github package URL (`https://maven.pkg.github.com/microsoft/*`) is wrong, no packages available at this time. Right? Can you pleas point me to the actual repository. Preferably Maven Central :)

Thanks,
-marc

**Comment by karianna** [comment]
Hi Marc,

Great to have you on board!  That URL should work, so in the POM file of _your_ project that uses GCToolKit you should add that repository as well as the three modules:

```
<repositories>
        ...
        <repository>
            <id>github</id>
            <name>GCToolKit packages</name>
            <url>https://maven.pkg.github.com/microsoft/*</url>
        </repository>
        ...
</repositories>

...

<dependencies>

...

        <dependency>
            <groupId>com.microsoft.gctoolkit</groupId>
            <artifactId>api</artifactId>
            <version>2.0.1</version>
        </dependency>
        
        <dependency>
            <groupId>com.microsoft.gctoolkit</groupId>
            <artifactId>parser</artifactId>
            <version>2.0.1</version>
        </dependency>
        
        <dependency>
            <groupId>com.microsoft.gctoolkit</groupId>
            <artifactId>vertx</artifactId>
            <version>2.0.1</version>
        </dependency>

...

</dependencies>

```

Hopefully, that should all resolve for you.

**Comment by marchof** [comment]
> Failed to execute goal on project gctoolkit-playground: Could not resolve dependencies for project com.mountainminds:gctoolkit-playground:jar:0.0.1-SNAPSHOT: Failed to collect dependencies at com.microsoft.gctoolkit:api:jar:2.0.1: Failed to read artifact descriptor for com.microsoft.gctoolkit:api:jar:2.0.1: Could not transfer artifact com.microsoft.gctoolkit:api:pom:2.0.1 from/to github (https://maven.pkg.github.com/microsoft/*): Access denied to https://maven.pkg.github.com/microsoft/*/com/microsoft/gctoolkit/api/2.0.1/api-2.0.1.pom. Error code 401, -> [Help 1]

Ah, seems to be an Github issue: You need to configure Maven with credentials to access artifacts. Will try later with my credentials.

Would be more convenient to have artifacts on Maven Central :)

**Comment by karianna** [comment]
> > Failed to execute goal on project gctoolkit-playground: Could not resolve dependencies for project com.mountainminds:gctoolkit-playground:jar:0.0.1-SNAPSHOT: Failed to collect dependencies at com.microsoft.gctoolkit:api:jar:2.0.1: Failed to read artifact descriptor for com.microsoft.gctoolkit:api:jar:2.0.1: Could not transfer artifact com.microsoft.gctoolkit:api:pom:2.0.1 from/to github (https://maven.pkg.github.com/microsoft/*): Access denied to https://maven.pkg.github.com/microsoft/*/com/microsoft/gctoolkit/api/2.0.1/api-2.0.1.pom. Error code 401, -> [Help 1]
> 
> Ah, seems to be an Github issue: You need to configure Maven with credentials to access artifacts. Will try later with my credentials.
> 
> Would be more convenient to have artifacts on Maven Central :)

Agreed but sadly it's not the sort of artifact Maven central would accept.

**Comment by marchof** [comment]
If I tried with PAT configured:

> [ERROR] Failed to execute goal on project gctoolkit-playground: Could not resolve dependencies for project com.mountainminds:gctoolkit-playground:jar:0.0.1-SNAPSHOT: Failed to collect dependencies at com.microsoft.gctoolkit:api:jar:2.0.1: Failed to read artifact descriptor for com.microsoft.gctoolkit:api:jar:2.0.1: Could not transfer artifact com.microsoft.gctoolkit:api:pom:2.0.1 from/to github (https://maven.pkg.github.com/microsoft/*): Failed to transfer https://maven.pkg.github.com/microsoft/*/com/microsoft/gctoolkit/api/2.0.1/api-2.0.1.pom. Error code 400, -> [Help 1]

Are you sure there is a release? The current [pom.xml](https://github.com/microsoft/gctoolkit/blob/main/pom.xml) only mentions `2.0.1-SNAPSHOT`.

**Comment by brunoborges** [comment]
This has been fixed with #90
